descriptors/ModuleDescriptor-template.json

```diff
@@ -2243,6 +2243,32 @@
       "description" : "Permission should be provided to reopen order"
     }
   ],
+  "metadata": {
+    "user": {
+      "type": "system",
+      "permissions": [
```

Comment:
please add also these permissions:
"organizations.organizations.collection.get"
"change-manager.jobExecutions.item.get"

these permissions are not required:
"invoice-storage.invoice-line-number.get",
"invoice-storage.invoice-lines.item.post",
"invoice-storage.invoice-lines.item.put",
"invoice-storage.invoice-lines.collection.get",
"invoice-storage.invoice-number.get",
"invoice-storage.invoices.item.get",
"invoice-storage.invoices.item.post",
"invoice-storage.invoices.item.put",

Reply:
@RuslanLavrov  Done! can you double-check please?